Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds.py

```diff
@@ -0,0 +1,223 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+''' IMPORTS '''
+
+
+import requests # noqa
+import traceback # noqa
+from typing import Dict, Any # noqa
+from datetime import datetime, timedelta # noqa
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+SNX_ID_KEY = "id"
+SNX_PATTERN_KEY = "pattern"
+SNX_OBJECTS_KEY = "objects"
+SNX_CREATED_KEY = "created"
+SNX_MODIFIED_KEY = "modified"
+SNX_DESCRIPTIONS_KEY = "description"
+SNX_FEED_KEY = "SecneurX Threat Feeds"
+DEMISO_DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+class Client(BaseClient):
+    """Implements class for SecneurX Threat Feeds."""
+    def get_feeds(self, paramsValue: Optional[str] = None):
```

Comment:
as the paramsValue refer to a date please change the name of the argument to a name that will reflect the meaning

Reply:
Okay